common/utils/src/main/resources/error/error-classes.json

```diff
@@ -451,6 +451,12 @@
     ],
     "sqlState" : "42704"
   },
+  "COLLATION_INVALID_NAME" : {
+    "message" : [
+      "The value <collationName> does not represent correct collation name."
+    ],
+    "sqlState" : "22018"
```

Comment:
Could you elaborate a little bit why did you chose the state:
```json
    "22018": {
        "description": "invalid character value for cast",
        "origin": "SQL/Foundation",
        "standard": "Y",
        "usedBy": ["SQL/Foundation", "PostgreSQL", "DB2", "Redshift", "Oracle", "SQL Server"]
    },
```

Reply:
Yeah, this probably is not the right error state. Waiting for your feedback on the appropriate one.

I just checked PGSql and it returns `"description": "An undefined column or parameter name was detected.",` Should we do the same?

Reply:
SGTM

Reply:
> I just checked PGSql and it returns "42704 - undefined_object Should we do the same?

The `42` SQLSTATE class seems reasonable: `"42": "Syntax Error or Access Rule Violation",`. The `42704` itself is a little bit generic but might be suitable.

Reply:
done